backend/Dockerfile

```diff
@@ -1,44 +1,48 @@
 # SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
 # SPDX-License-Identifier: Apache-2.0
 
-ARG BASE_IMAGE=python:3.12-slim-bookworm
+ARG BASE_IMAGE=debian:bookworm-slim
 FROM $BASE_IMAGE
 
 SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
 ENV SHELL=/bin/bash
 
+EXPOSE 8000
+
 USER root
 
-RUN apt-get update && \
-    apt-get upgrade --yes && \
-    apt-get install --yes \
+ADD https://astral.sh/uv/install.sh /uv-installer.sh
+ENV UV_UNMANAGED_INSTALL=/usr/bin/
+ENV UV_PYTHON_INSTALL_DIR=/etc/python
```

Comment:
Is there a chance we'd be able to install uv in our restricted environment using `COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/`? That way, we wouldn't need to install curl and it seems more robust than using the install script.

Reply:
No, unfortunately not. `ghrc.io` is not available from certain build environments, so we would need to make it configurable. Arguments in the `--from` parameter are not supported by Docker unfortunately.